addons/web_editor/static/src/components/upload_progress_toast/upload_service.js

```diff
@@ -98,8 +98,8 @@ export const uploadService = {
                 // limited by bandwidth.
                 for (const sortedFile of sortedFiles) {
                     const file = progressToast.files[sortedFile.progressToastId];
-                    const dataURL = await getDataURLFromFile(sortedFile);
                     try {
+                        const dataURL = await getDataURLFromFile(sortedFile);
```

Comment:
Do you have an use case for the fix in this file?

Reply:
As mentioned above, it will be having 'null' value for data parameter so, to handle this it is putted in try-catch block.